java/openmldb-taskmanager/src/test/java/com/_4paradigm/openmldb/taskmanager/TaskManagerClient_Test.java

```diff
@@ -0,0 +1,23 @@
+package com._4paradigm.openmldb.taskmanager;
+
+import com._4paradigm.openmldb.taskmanager.client.TaskManagerClient;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+
+public abstract class TaskManagerClient_Test {
```

Comment:
no underscore, Keep in mind the class name convention

Reply:
I get it